packages/bun/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@sentry/bun",
+  "version": "7.69.0",
+  "description": "Official Sentry SDK for bun",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/bun",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "main": "build/esm/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/types/index.d.ts",
+  "typesVersions": {
+    "<4.9": {
+      "build/npm/types/index.d.ts": [
+        "build/npm/types-ts3.8/index.d.ts"
+      ]
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/core": "7.69.0",
+    "@sentry/node": "7.69.0",
+    "@sentry/types": "7.69.0",
+    "@sentry/utils": "7.69.0"
+  },
+  "devDependencies": {
+    "bun-types": "latest"
```

Comment:
I wonder how we can say what versions of Bun we support 🤔

Reply:
I wouldn't bother to much, let's lock in once we know 😬